src/vs/code/electron-sandbox/processExplorer/processExplorerMain.ts

```diff
@@ -26,8 +26,8 @@ import { applyZoom, zoomIn, zoomOut } from 'vs/platform/window/electron-sandbox/
 import { StandardKeyboardEvent } from 'vs/base/browser/keyboardEvent';
 import { KeyCode } from 'vs/base/common/keyCodes';
 
-const DEBUG_FLAGS_PATTERN = /\s--(inspect|debug)(-brk|port)?=(\d+)?/;
-const DEBUG_PORT_PATTERN = /\s--(inspect|debug)-port=(\d+)/;
+const DEBUG_FLAGS_PATTERN = /\s--inspect(?:-brk|port)?=(\d+)?/;
+const DEBUG_PORT_PATTERN = /\s--inspect-port=(\d+)/;
```

Comment:
I'm all in on capture groups now for readability, they made terminal links so much easier to deal with. You could make this `(?<port>\d+)` and then access via `matches?.groups?.port`